fdbserver/include/fdbserver/DDTeamCollection.h

```diff
@@ -328,6 +330,9 @@ class DDTeamCollection : public ReferenceCounted<DDTeamCollection> {
 	// When "wantHealthy" is true, only return if the team is healthy.
 	Optional<Reference<IDataDistributionTeam>> findTeamFromServers(const std::vector<UID>& servers, bool wantHealthy);
 
+	// Evaluate the CPU and AvailableSpace of source team.
```

Comment:
We could add a bit more clarification to the comment here, something like: ".... to determine if a shard can remain on the source team rather than being moved to a new destination team".